internal/signature/signer.go

```diff
@@ -62,6 +63,14 @@ func Sign(ctx context.Context, req *proto.GenerateSignatureRequest) (*proto.Gene
 		}
 	}
 
+	// Map of Notation RSA-OAEP hash algorithms to Vault's transit engine /sign endpoint equivalent
```

Comment:
This isn't correct. RSA-OAEP is an encryption algorithm; RSA-PSS is used for signing. 

More correctly, I think this would be commented as Notary to OpenBao/Vault hash algorithm naming conversion map. My 2c.